src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/Utilities/TestDataHelper.cs

```diff
@@ -106,26 +106,18 @@ internal static long AddProvider(long ukprn)
                 var censusDate = startDate.LastDayOfMonth();
                 var period = 1;
 
-                transactionTypes = transactionTypes ?? new[] { TransactionType.Balancing, TransactionType.Completion, TransactionType.Learning };
 
                 while (censusDate <= endDate && period <= 12)
                 {
-                    foreach (var traxType in transactionTypes)
-                    {
-                        AddPriceEpisodePeriodMatch(id, ukprn, learnerRefNumber, aimSequenceNumber, priceEpisodeIdentifier, period, passedDataLock, notPayablePeriods, notMatchedPeriods, traxType);
-                    }
-
+                    
+                    AddPriceEpisodePeriodMatch(id, ukprn, learnerRefNumber, aimSequenceNumber, priceEpisodeIdentifier, period, passedDataLock, notPayablePeriods, notMatchedPeriods, transactionTypesFlag);
```

Comment:
Does this do the same as the previous version where it ran for each transaction type or is that now not required as a result?